pkg/services/rpcsrv/server_test.go

```diff
@@ -74,22 +74,22 @@ type rpcTestCase struct {
 }
 
 const genesisBlockHash = "0f8fb4e17d2ab9f3097af75ca7fd16064160fb8043db94909e00dd4e257b9dc4"
-const testContractHash = "565cff9508ebc75aadd7fe59f38dac610ab6093c"
-const deploymentTxHash = "a14390941cc3a1d87393eff720a722e9cd350bd6ed233c5fe2001326c80eb68e"
+const testContractHash = "449fe8fbd4523072f5e3a4dfa17a494c119d4c08"
+const deploymentTxHash = "bbb8ec059dd320dc9de5a8fb8c75351d8e369fca0256f7a6bdb623dcf71861ed"
 
 const (
 	verifyContractHash                = "06ed5314c2e4cb103029a60b86d46afa2fb8f67c"
 	verifyContractAVM                 = "VwIAQS1RCDBwDBTunqIsJ+NL0BSPxBCOCPdOj1BIskrZMCQE2zBxaBPOStkoJATbKGlK2SgkBNsol0A="
-	verifyWithArgsContractHash        = "4dc916254efd2947c93b11207e8ffc0bb56161c5"
-	nnsContractHash                   = "892429fcd47c30f8451781acc627e8b20e0d64f3"
+	verifyWithArgsContractHash        = "6261b3bf753bdc3d24c1327a23fd891e1c8a7ccd"
+	nnsContractHash                   = "450d1918a72fef97b48096bfec8d749961deef55"
 	nnsToken1ID                       = "6e656f2e636f6d"
-	nfsoContractHash                  = "730ebe719ab8e3b69d11dafc95cdb9bf409db179"
+	nfsoContractHash                  = "2f5c1826bb4da1c764a8871427e4044cf3e82dbd"
 	nfsoToken1ID                      = "7e244ffd6aa85fb1579d2ed22e9b761ab62e3486"
 	storageContractHash               = "ebc0c16a76c808cd4dde6bcc063f09e45e331ec7"
 	faultedTxHashLE                   = "82279bfe9bada282ca0f8cb8e0bb124b921af36f00c69a518320322c6f4fef60"
 	faultedTxBlock             uint32 = 23
 	invokescriptContractAVM           = "VwIADBQBDAMOBQYMDQIODw0DDgcJAAAAAErZMCQE2zBwaEH4J+yMqiYEEUAMFA0PAwIJAAIBAwcDBAUCAQAOBgwJStkwJATbMHFpQfgn7IyqJgQSQBNA"
-	block20StateRootLE                = "858c873539d6d24a70f2be13f9dafc61aef2b63c2aa16bb440676de6e44e3cf1"
+	block20StateRootLE                = "7e411d227a41b760e3c0309e12eb35c01064ffec59d237fcc495a69e97d8c033"
```

Comment:
Still strange because in TestCreateBasicChain logs `Block #20 stateroot LE: 9a431f425932ffb5c0a9b569fae2d5bc086753de21b6a83d6a41b46aaf21d014`

```
GOROOT=/Users/ekaterinapavlova/go/go1.22.6 #gosetup
GOPATH=/Users/ekaterinapavlova/go #gosetup
/Users/ekaterinapavlova/go/go1.22.6/bin/go test -c -o /Users/ekaterinapavlova/Library/Caches/JetBrains/GoLand2024.2/tmp/GoLand/___TestCreateBasicChain_in_github_com_nspcc_dev_neo_go_pkg_core.test github.com/nspcc-dev/neo-go/pkg/core #gosetup
/Users/ekaterinapavlova/go/go1.22.6/bin/go tool test2json -t /Users/ekaterinapavlova/Library/Caches/JetBrains/GoLand2024.2/tmp/GoLand/___TestCreateBasicChain_in_github_com_nspcc_dev_neo_go_pkg_core.test -test.v=test2json -test.paniconexit0 -test.run ^\QTestCreateBasicChain\E$
=== RUN   TestCreateBasicChain
    logger.go:146: 2024-11-14T15:52:24.190+0300	INFO	initial gas supply is not set or wrong, setting default value	{"InitialGASSupply": "52000000"}
    logger.go:146: 2024-11-14T15:52:24.191+0300	INFO	mempool size is not set or wrong, setting default value	{"MemPoolSize": 50000}
    logger.go:146: 2024-11-14T15:52:24.191+0300	INFO	P2PNotaryRequestPayloadPool size is not set or wrong, setting default value	{"P2PNotaryRequestPayloadPoolSize": 1000}
    logger.go:146: 2024-11-14T15:52:24.191+0300	INFO	MaxBlockSize is not set or wrong, setting default value	{"MaxBlockSize": 262144}
    logger.go:146: 2024-11-14T15:52:24.191+0300	INFO	MaxBlockSystemFee is not set or wrong, setting default value	{"MaxBlockSystemFee": 900000000000}
    logger.go:146: 2024-11-14T15:52:24.191+0300	INFO	MaxTransactionsPerBlock is not set or wrong, using default value	{"MaxTransactionsPerBlock": 512}
    logger.go:146: 2024-11-14T15:52:24.191+0300	INFO	MaxValidUntilBlockIncrement is not set or wrong, using default value	{"MaxValidUntilBlockIncrement": 86400}
    logger.go:146: 2024-11-14T15:52:24.191+0300	INFO	Hardforks are not set, using default value
    logger.go:146: 2024-11-14T15:52:24.194+0300	INFO	no storage version found! creating genesis block
    basic.go:98: native GAS hash: cf76e28bd0062c4a478ee35561011319f3cfa4d2
    basic.go:99: native NEO hash: f563ea40bc283d4d0e05c48ea305b3f2a07340ef
    basic.go:100: native Policy hash: 7bc681c0a1f71d543457b68bba8d5f9fdd4e5ecc
    basic.go:101: native Notary hash: 3bec3531119bbad76dd044920b0de6c3194fe1c1
    basic.go:102: Block0 hash: 0f8fb4e17d2ab9f3097af75ca7fd16064160fb8043db94909e00dd4e257b9dc4
    logger.go:146: 2024-11-14T15:52:24.196+0300	DEBUG	done processing headers	{"headerIndex": 1, "blockHeight": 0, "took": "348µs"}
    basic.go:128: Block1 hash: f42e2ae74bbea6aa1789fdc4efa35ad55b04335442637c091eafb5b0e779dae7
    basic.go:134: Block1 base64: AAAAAMSdeyVO3QCekJTbQ4D7YEEGFv2nXPd6CfO5Kn3htI8PX/PprXpQAEFi6itdbZ/kpCb+8quS+HR6yLRQ4zC9VQuJ6hnvVQEAAAAAAAAAAAAAAQAAAABouegejA5skHm0udEF6HinbT8iPwHGDECwNqq8Bt1tNYnCT6gXNNIsRaNGlW3Ys8B2LO9Nh9bgwmT6NAP8KM+kjUCqB0avprTRvgsIzmVhXqEjgoL3K+9XDEDojqNsOJJrJqq4dy8DyABuBWre52T+OaoE0TSzaRISaA5pzotF1Zuswd321t/NKGCUwKA1fsRFVeDxiWsfQe6MDEBsDlIkkuqFCSZFjmBm/0a+gzZYlXcUPwJteeir8mzEt6zj34N8WI4h3NVvZcfqn1ZLKO/oDk4hEMDzZTkWk82DkxMMIQIQOn990BZVhZf3lg0nxRakOU/ZaLnmUVXrSwE+QEBAbgwhAqe8Vf6GhOARl2jRBLoweVvcyGYZ6GSt0mFWcj7Rhc1iDCECs2Ir9AF73+MXxYrtX0x1PyBrfbiWBG+n13S7xL9/jcIMIQPZDAffY+aQzneRLhCrUazJRLZoYCN7YIxPj4MJ5x7mmRRBntDcOgIAAQAAAJY/mAAAAAAAJHVDAAAAAAABAAAAAWi56B6MDmyQebS50QXoeKdtPyI/gABaCwIY3fUFDBTunqIsJ+NL0BSPxBCOCPdOj1BIsgwUaLnoHowObJB5tLnRBeh4p20/Ij8UwB8MCHRyYW5zZmVyDBT1Y+pAvCg9TQ4FxI6jBbPyoHNA70FifVtSAcYMQIlR3oYUxUdOWkbW6audp43+/yBO3sx+giqyd4dO/DsTR0ZNEDJYl8mH+Kd5I7LG5oS/nu+76NnTgAvsZ8kHVz0MQMn9qQTsW4iIV4lkZD57WsKIctqt4YmKBl3VefAfunik+kI4IcbDnOayJVTUM/H8ZNNhcOTFuQv1agJ/m5FKF9kMQPWQxmcI46YB6w6IWN44UU2GyNAPPcyY5x90uTmWmHKd3Oc8oH7Lg5GFiNEUk10W2zZsT0lQWvxkY364Tvi2yJKTEwwhAhA6f33QFlWFl/eWDSfFFqQ5T9loueZRVetLAT5AQEBuDCECp7xV/oaE4BGXaNEEujB5W9zIZhnoZK3SYVZyPtGFzWIMIQKzYiv0AXvf4xfFiu1fTHU/IGt9uJYEb6fXdLvEv3+NwgwhA9kMB99j5pDOd5EuEKtRrMlEtmhgI3tgjE+PgwnnHuaZFEGe0Nw6AAIAAACWP5gAAAAAAMSEQwAAAAAAAQAAAAFouegejA5skHm0udEF6HinbT8iP4AAXgsDAOh2SBcAAAAMFO6eoiwn40vQFI/EEI4I906PUEiyDBRouegejA5skHm0udEF6HinbT8iPxTAHwwIdHJhbnNmZXIMFM924ovQBixKR47jVWEBExnzz6TSQWJ9W1IBxgxAagNXvPCnXPgu3HmWo7TSyDGawR6o6kpQO2RU9Qm7PfPRA3iyp72SH1qqnFK2SgqtdsdBADr8fhaycWKcnzs0sAxAM6QkmIl1FjAs4sUVjvobE+qLsUsRs63lv8o9OUvWTtpNJcCuWWTXFVULKvn5jEfMHOOfNwUdQlByINO21EakdgxAYg40QKKGJYsazRUliaGAtgFcN3KRdsd9imKvxJW3neEpllTGo386XUdI4ddWP6x3izrRbkIfmDr92EF3Pc8vB5MTDCECEDp/fdAWVYWX95YNJ8UWpDlP2Wi55lFV60sBPkBAQG4MIQKnvFX+hoTgEZdo0QS6MHlb3MhmGehkrdJhVnI+0YXNYgwhArNiK/QBe9/jF8WK7V9MdT8ga324lgRvp9d0u8S/f43CDCED2QwH32PmkM53kS4Qq1GsyUS2aGAje2CMT4+DCece5pkUQZ7Q3Do=
    basic.go:135: Block1 JSON: {"hash":"0xf42e2ae74bbea6aa1789fdc4efa35ad55b04335442637c091eafb5b0e779dae7","version":0,"previousblockhash":"0x0f8fb4e17d2ab9f3097af75ca7fd16064160fb8043db94909e00dd4e257b9dc4","merkleroot":"0x0b55bd30e350b4c87a74f892abf2fe26a4e49f6d5d2bea624100507aade9f35f","time":1468595301001,"nonce":"0000000000000000","index":1,"nextconsensus":"NVTiAjNgagDkTr5HTzDmQP9kPwPHN5BgVq","primary":0,"witnesses":[{"invocation":"DECwNqq8Bt1tNYnCT6gXNNIsRaNGlW3Ys8B2LO9Nh9bgwmT6NAP8KM+kjUCqB0avprTRvgsIzmVhXqEjgoL3K+9XDEDojqNsOJJrJqq4dy8DyABuBWre52T+OaoE0TSzaRISaA5pzotF1Zuswd321t/NKGCUwKA1fsRFVeDxiWsfQe6MDEBsDlIkkuqFCSZFjmBm/0a+gzZYlXcUPwJteeir8mzEt6zj34N8WI4h3NVvZcfqn1ZLKO/oDk4hEMDzZTkWk82D","verification":"EwwhAhA6f33QFlWFl/eWDSfFFqQ5T9loueZRVetLAT5AQEBuDCECp7xV/oaE4BGXaNEEujB5W9zIZhnoZK3SYVZyPtGFzWIMIQKzYiv0AXvf4xfFiu1fTHU/IGt9uJYEb6fXdLvEv3+NwgwhA9kMB99j5pDOd5EuEKtRrMlEtmhgI3tgjE+PgwnnHuaZFEGe0Nw6"}],"tx":[{"hash":"0xf01e1986ee63cc9437516745cfb7f57108432d5493a7c159a1082900c6afb316","size":487,"version":0,"nonce":1,"sender":"NVTiAjNgagDkTr5HTzDmQP9kPwPHN5BgVq","sysfee":"9977750","netfee":"4420900","validuntilblock":1,"attributes":[],"signers":[{"account":"0x3f223f6da778e805d1b9b479906c0e8c1ee8b968","scopes":"Global"}],"script":"CwIY3fUFDBTunqIsJ+NL0BSPxBCOCPdOj1BIsgwUaLnoHowObJB5tLnRBeh4p20/Ij8UwB8MCHRyYW5zZmVyDBT1Y+pAvCg9TQ4FxI6jBbPyoHNA70FifVtS","witnesses":[{"invocation":"DECJUd6GFMVHTlpG1umrnaeN/v8gTt7MfoIqsneHTvw7E0dGTRAyWJfJh/ineSOyxuaEv57vu+jZ04AL7GfJB1c9DEDJ/akE7FuIiFeJZGQ+e1rCiHLareGJigZd1XnwH7p4pPpCOCHGw5zmsiVU1DPx/GTTYXDkxbkL9WoCf5uRShfZDED1kMZnCOOmAesOiFjeOFFNhsjQDz3MmOcfdLk5lphyndznPKB+y4ORhYjRFJNdFts2bE9JUFr8ZGN+uE74tsiS","verification":"EwwhAhA6f33QFlWFl/eWDSfFFqQ5T9loueZRVetLAT5AQEBuDCECp7xV/oaE4BGXaNEEujB5W9zIZhnoZK3SYVZyPtGFzWIMIQKzYiv0AXvf4xfFiu1fTHU/IGt9uJYEb6fXdLvEv3+NwgwhA9kMB99j5pDOd5EuEKtRrMlEtmhgI3tgjE+PgwnnHuaZFEGe0Nw6"}]},{"hash":"0x8b9dbed5cb098227175837a95a765f6ad92d50e1be3b6cb84ec47f6e3c5504ac","size":491,"version":0,"nonce":2,"sender":"NVTiAjNgagDkTr5HTzDmQP9kPwPHN5BgVq","sysfee":"9977750","netfee":"4424900","validuntilblock":1,"attributes":[],"signers":[{"account":"0x3f223f6da778e805d1b9b479906c0e8c1ee8b968","scopes":"Global"}],"script":"CwMA6HZIFwAAAAwU7p6iLCfjS9AUj8QQjgj3To9QSLIMFGi56B6MDmyQebS50QXoeKdtPyI/FMAfDAh0cmFuc2ZlcgwUz3bii9AGLEpHjuNVYQETGfPPpNJBYn1bUg==","witnesses":[{"invocation":"DEBqA1e88Kdc+C7ceZajtNLIMZrBHqjqSlA7ZFT1Cbs989EDeLKnvZIfWqqcUrZKCq12x0EAOvx+FrJxYpyfOzSwDEAzpCSYiXUWMCzixRWO+hsT6ouxSxGzreW/yj05S9ZO2k0lwK5ZZNcVVQsq+fmMR8wc4583BR1CUHIg07bURqR2DEBiDjRAooYlixrNFSWJoYC2AVw3cpF2x32KYq/Elbed4SmWVMajfzpdR0jh11Y/rHeLOtFuQh+YOv3YQXc9zy8H","verification":"EwwhAhA6f33QFlWFl/eWDSfFFqQ5T9loueZRVetLAT5AQEBuDCECp7xV/oaE4BGXaNEEujB5W9zIZhnoZK3SYVZyPtGFzWIMIQKzYiv0AXvf4xfFiu1fTHU/IGt9uJYEb6fXdLvEv3+NwgwhA9kMB99j5pDOd5EuEKtRrMlEtmhgI3tgjE+PgwnnHuaZFEGe0Nw6"}]}]}
    basic.go:141: Header1 base64: AAAAAMSdeyVO3QCekJTbQ4D7YEEGFv2nXPd6CfO5Kn3htI8PX/PprXpQAEFi6itdbZ/kpCb+8quS+HR6yLRQ4zC9VQuJ6hnvVQEAAAAAAAAAAAAAAQAAAABouegejA5skHm0udEF6HinbT8iPwHGDECwNqq8Bt1tNYnCT6gXNNIsRaNGlW3Ys8B2LO9Nh9bgwmT6NAP8KM+kjUCqB0avprTRvgsIzmVhXqEjgoL3K+9XDEDojqNsOJJrJqq4dy8DyABuBWre52T+OaoE0TSzaRISaA5pzotF1Zuswd321t/NKGCUwKA1fsRFVeDxiWsfQe6MDEBsDlIkkuqFCSZFjmBm/0a+gzZYlXcUPwJteeir8mzEt6zj34N8WI4h3NVvZcfqn1ZLKO/oDk4hEMDzZTkWk82DkxMMIQIQOn990BZVhZf3lg0nxRakOU/ZaLnmUVXrSwE+QEBAbgwhAqe8Vf6GhOARl2jRBLoweVvcyGYZ6GSt0mFWcj7Rhc1iDCECs2Ir9AF73+MXxYrtX0x1PyBrfbiWBG+n13S7xL9/jcIMIQPZDAffY+aQzneRLhCrUazJRLZoYCN7YIxPj4MJ5x7mmRRBntDcOg==
    basic.go:142: Header1 JSON: {"hash":"0xf42e2ae74bbea6aa1789fdc4efa35ad55b04335442637c091eafb5b0e779dae7","version":0,"previousblockhash":"0x0f8fb4e17d2ab9f3097af75ca7fd16064160fb8043db94909e00dd4e257b9dc4","merkleroot":"0x0b55bd30e350b4c87a74f892abf2fe26a4e49f6d5d2bea624100507aade9f35f","time":1468595301001,"nonce":"0000000000000000","index":1,"nextconsensus":"NVTiAjNgagDkTr5HTzDmQP9kPwPHN5BgVq","primary":0,"witnesses":[{"invocation":"DECwNqq8Bt1tNYnCT6gXNNIsRaNGlW3Ys8B2LO9Nh9bgwmT6NAP8KM+kjUCqB0avprTRvgsIzmVhXqEjgoL3K+9XDEDojqNsOJJrJqq4dy8DyABuBWre52T+OaoE0TSzaRISaA5pzotF1Zuswd321t/NKGCUwKA1fsRFVeDxiWsfQe6MDEBsDlIkkuqFCSZFjmBm/0a+gzZYlXcUPwJteeir8mzEt6zj34N8WI4h3NVvZcfqn1ZLKO/oDk4hEMDzZTkWk82D","verification":"EwwhAhA6f33QFlWFl/eWDSfFFqQ5T9loueZRVetLAT5AQEBuDCECp7xV/oaE4BGXaNEEujB5W9zIZhnoZK3SYVZyPtGFzWIMIQKzYiv0AXvf4xfFiu1fTHU/IGt9uJYEb6fXdLvEv3+NwgwhA9kMB99j5pDOd5EuEKtRrMlEtmhgI3tgjE+PgwnnHuaZFEGe0Nw6"}]}
    basic.go:145: txMoveNeo hash: f01e1986ee63cc9437516745cfb7f57108432d5493a7c159a1082900c6afb316
    basic.go:146: txMoveNeo JSON: {"hash":"0xf01e1986ee63cc9437516745cfb7f57108432d5493a7c159a1082900c6afb316","size":487,"version":0,"nonce":1,"sender":"NVTiAjNgagDkTr5HTzDmQP9kPwPHN5BgVq","sysfee":"9977750","netfee":"4420900","validuntilblock":1,"attributes":[],"signers":[{"account":"0x3f223f6da778e805d1b9b479906c0e8c1ee8b968","scopes":"Global"}],"script":"CwIY3fUFDBTunqIsJ+NL0BSPxBCOCPdOj1BIsgwUaLnoHowObJB5tLnRBeh4p20/Ij8UwB8MCHRyYW5zZmVyDBT1Y+pAvCg9TQ4FxI6jBbPyoHNA70FifVtS","witnesses":[{"invocation":"DECJUd6GFMVHTlpG1umrnaeN/v8gTt7MfoIqsneHTvw7E0dGTRAyWJfJh/ineSOyxuaEv57vu+jZ04AL7GfJB1c9DEDJ/akE7FuIiFeJZGQ+e1rCiHLareGJigZd1XnwH7p4pPpCOCHGw5zmsiVU1DPx/GTTYXDkxbkL9WoCf5uRShfZDED1kMZnCOOmAesOiFjeOFFNhsjQDz3MmOcfdLk5lphyndznPKB+y4ORhYjRFJNdFts2bE9JUFr8ZGN+uE74tsiS","verification":"EwwhAhA6f33QFlWFl/eWDSfFFqQ5T9loueZRVetLAT5AQEBuDCECp7xV/oaE4BGXaNEEujB5W9zIZhnoZK3SYVZyPtGFzWIMIQKzYiv0AXvf4xfFiu1fTHU/IGt9uJYEb6fXdLvEv3+NwgwhA9kMB99j5pDOd5EuEKtRrMlEtmhgI3tgjE+PgwnnHuaZFEGe0Nw6"}]}
    basic.go:147: txMoveNeo base64: AAEAAACWP5gAAAAAACR1QwAAAAAAAQAAAAFouegejA5skHm0udEF6HinbT8iP4AAWgsCGN31BQwU7p6iLCfjS9AUj8QQjgj3To9QSLIMFGi56B6MDmyQebS50QXoeKdtPyI/FMAfDAh0cmFuc2ZlcgwU9WPqQLwoPU0OBcSOowWz8qBzQO9BYn1bUgHGDECJUd6GFMVHTlpG1umrnaeN/v8gTt7MfoIqsneHTvw7E0dGTRAyWJfJh/ineSOyxuaEv57vu+jZ04AL7GfJB1c9DEDJ/akE7FuIiFeJZGQ+e1rCiHLareGJigZd1XnwH7p4pPpCOCHGw5zmsiVU1DPx/GTTYXDkxbkL9WoCf5uRShfZDED1kMZnCOOmAesOiFjeOFFNhsjQDz3MmOcfdLk5lphyndznPKB+y4ORhYjRFJNdFts2bE9JUFr8ZGN+uE74tsiSkxMMIQIQOn990BZVhZf3lg0nxRakOU/ZaLnmUVXrSwE+QEBAbgwhAqe8Vf6GhOARl2jRBLoweVvcyGYZ6GSt0mFWcj7Rhc1iDCECs2Ir9AF73+MXxYrtX0x1PyBrfbiWBG+n13S7xL9/jcIMIQPZDAffY+aQzneRLhCrUazJRLZoYCN7YIxPj4MJ5x7mmRRBntDcOg==
    basic.go:148: txMoveGas hash: 8b9dbed5cb098227175837a95a765f6ad92d50e1be3b6cb84ec47f6e3c5504ac
    basic.go:152: header hash: f42e2ae74bbea6aa1789fdc4efa35ad55b04335442637c091eafb5b0e779dae7
    basic.go:155: header: 00000000c49d7b254edd009e9094db4380fb60410616fda75cf77a09f3b92a7de1b48f0f5ff3e9ad7a50004162ea2b5d6d9fe4a426fef2ab92f8747ac8b450e330bd550b89ea19ef550100000000000000000000010000000068b9e81e8c0e6c9079b4b9d105e878a76d3f223f01c60c40b036aabc06dd6d3589c24fa81734d22c45a346956dd8b3c0762cef4d87d6e0c264fa3403fc28cfa48d40aa0746afa6b4d1be0b08ce65615ea1238282f72bef570c40e88ea36c38926b26aab8772f03c8006e056adee764fe39aa04d134b3691212680e69ce8b45d59bacc1ddf6d6dfcd286094c0a0357ec44555e0f1896b1f41ee8c0c406c0e522492ea850926458e6066ff46be8336589577143f026d79e8abf26cc4b7ace3df837c588e21dcd56f65c7ea9f564b28efe80e4e2110c0f365391693cd8393130c2102103a7f7dd016558597f7960d27c516a4394fd968b9e65155eb4b013e4040406e0c2102a7bc55fe8684e0119768d104ba30795bdcc86619e864add26156723ed185cd620c2102b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc20c2103d90c07df63e690ce77912e10ab51acc944b66860237b608c4f8f8309e71ee69914419ed0dc3a
    basic.go:300: contract (../../internal/basicchain/testdata/test_contract.go): 
        	Hash: 449fe8fbd4523072f5e3a4dfa17a494c119d4c08
        	AVM: VgEMFK7Y2ztdZkygS9177TeZL/lYVgtBStkwJATbMGBAVwQAQZv2Z85wQdv+qHRxAkBCDwByaGlqU0HmPxiEC2lqUxPAcwwIVHJhbnNmZXJrUEGVAW9hCEBXAwNB2/6odHBoNwAAcTcBAABkMExoWGkLVDcCAHJqqiYaDBVvd25lciB0cmFuc2ZlciBmYWlsZWQ6aFgQC1Q3AgByaqomGgwVb3duZXIgdHJhbnNmZXIgZmFpbGVkOkAIQFcGBEGb9mfOcHhK2CQFyiIERRAAFCghDBZpbnZhbGlkICdmcm9tJyBhZGRyZXNzQc/nR5YJQHlK2CQFyiIERRAAFCgfDBRpbnZhbGlkICd0bycgYWRkcmVzc0HP50eWCUB6EC4ZDA5pbnZhbGlkIGFtb3VudEHP50eWCUAQcWh4UEGSXegxcmrYJAtqStkhJATbIXFpei4dDBJpbnN1ZmZpY2llbnQgZnVuZHNBz+dHlglAaXqfcWh4aVNB5j8YhBBzaHlQQZJd6DFyatgkC2pK2SEkBNshc2t6nnNoeWtTQeY/GIR4eXpTE8B0DAhUcmFuc2ZlcmxQQZUBb2F5NwMA2CQjeHp7UxPAdXkMDm9uTkVQMTdQYXltZW50H21UQWJ9W1JFCEBXAwFBm/ZnznB4StgkBcoiBEUQABQoFAwPaW52YWxpZCBhZGRyZXNzOhBxaHhQQZJd6DFyatgkC2pK2SEkBNshcWlADANSVUJAEkACQEIPAEBXAQJBm/ZnznBoeHlTQeY/GIQIQA==
    logger.go:146: 2024-11-14T15:52:24.280+0300	DEBUG	done processing headers	{"headerIndex": 2, "blockHeight": 1, "took": "255.458µs"}
    basic.go:160: txDeploy: bbb8ec059dd320dc9de5a8fb8c75351d8e369fca0256f7a6bdb623dcf71861ed
    basic.go:161: Block2 hash: fcda45a5748d1c5fbf31f0004641d863791419bafbd33e3a98f72389e7a0e4dd
    logger.go:146: 2024-11-14T15:52:24.281+0300	DEBUG	done processing headers	{"headerIndex": 3, "blockHeight": 2, "took": "228.125µs"}
    basic.go:166: txInv: 00f9d0337f572077101b483a7393897395f51b8e02dc22237f45a061a764fdcd
    logger.go:146: 2024-11-14T15:52:24.281+0300	DEBUG	done processing headers	{"headerIndex": 4, "blockHeight": 3, "took": "218.833µs"}
    logger.go:146: 2024-11-14T15:52:24.282+0300	DEBUG	done processing headers	{"headerIndex": 5, "blockHeight": 4, "took": "221.791µs"}
    basic.go:178: receiveRublesTx: 7b095a6f89ec6b4c78db920837adc8f3dc7a2e379462ca78157f0237d1e62bd3
    logger.go:146: 2024-11-14T15:52:24.283+0300	DEBUG	done processing headers	{"headerIndex": 6, "blockHeight": 5, "took": "209.208µs"}
    basic.go:182: sendRublesTx: 5585116f92f1a7779e8effd77ba9107008257ff1be3833ae649bacf63e75f333
    basic.go:300: contract (../../internal/basicchain/testdata/verify/verification_contract.go): 
        	Hash: 06ed5314c2e4cb103029a60b86d46afa2fb8f67c
        	AVM: VwIAQS1RCDBwDBTunqIsJ+NL0BSPxBCOCPdOj1BIskrZMCQE2zBxaBPOStkoJATbKGlK2SgkBNsol0A=
    logger.go:146: 2024-11-14T15:52:24.314+0300	DEBUG	done processing headers	{"headerIndex": 7, "blockHeight": 6, "took": "240.792µs"}
    logger.go:146: 2024-11-14T15:52:24.315+0300	DEBUG	done processing headers	{"headerIndex": 8, "blockHeight": 7, "took": "228.583µs"}
    basic.go:191: notaryDepositTxPriv0: c409e019431f5cdafcfe91af69d445ffed46e5bf5f599c77cbc038708fc3623f
    logger.go:146: 2024-11-14T15:52:24.316+0300	DEBUG	done processing headers	{"headerIndex": 9, "blockHeight": 8, "took": "239.916µs"}
    basic.go:199: Designated Notary node: 039b93e66ce7d53e66174d2556f74dd79ce8613414ec17da0395562eb081ffba2d
    basic.go:300: contract (../../internal/basicchain/testdata/verify_args/verification_with_args_contract.go): 
        	Hash: 6261b3bf753bdc3d24c1327a23fd891e1c8a7ccd
        	AVM: VwEDeAwLZ29vZF9zdHJpbmeXJAt5FSgHegiXIgMIcGhA
    logger.go:146: 2024-11-14T15:52:24.344+0300	DEBUG	done processing headers	{"headerIndex": 10, "blockHeight": 9, "took": "251.084µs"}
    basic.go:300: contract (../../examples/nft-nd-nns): 
        	Hash: 450d1918a72fef97b48096bfec8d749961deef55
        	AVM: VwECeSYDQEGb9mfOcGgMAQBK2TAkBNswEFNB5j8YhGgMARBK2TAkBNswAgDKmjtTQeY/GIRAVwABQbfDiAN4Es4wFQwQbmFtZSBoYXMgZXhwaXJlZDpAVwABeBDOQfgn7IwmA0B4E87YJA14E85B+CfsjKomGwwWbm90IHdpdG5lc3NlZCBieSBhZG1pbjpAVwECNRkKAAB5StkwJATbMHB4aFA3AABADANOTlNAEEBXAQBB9rRr4nBoNUYIAABAVwIBQfa0a+JwaHhQNXgIAABxaRDOQFcCAUH2tGvicGh4UDViCAAAcWkTzgwFYWRtaW5pEs4MCmV4cGlyYXRpb25pEc4MBG5hbWUTvkBXAgF4NYMJAACqJhIMDWludmFsaWQgb3duZXI6Qfa0a+JwaAwBAUrZMCQE2zBK2CYFRQwAeItQQZJd6DFxadgmBBBAaUrZISQE2yFAVwEAQfa0a+JwaAwBIUrZMCQE2zAALFNB3zC4mkBXAQF4NRoJAACqJhIMDWludmFsaWQgb3duZXI6Qfa0a+JwaAwBAkrZMCQE2zBK2CYFRQwAeIsUU0HfMLiaQFcFA3g13AgAAKomFQwQaW52YWxpZCByZWNlaXZlcjp5NWgHAABwQZv2Z85xaWhQNW4HAAByahDOc2tB+CfsjKomBAlAa3Rs2CYGeNgiAwkkIGzYJBl42CQVbErZKCQE2yh4StkoJATbKJciAwkiAwiqJi94ahBR0AtqE1HQaWhqwkpRz9RTNYcHAABpeWsRm1Q17wUAAGl5eBFUNeUFAABreHl6VDVhBgAACEBXAgE1SQgAAHgIUDWrCAAAqiYYDBNpbnZhbGlkIHJvb3QgZm9ybWF0OkGb9mfOcAwBIErZMCQE2zBK2CYFRQwAeErZMCQE2zCLcWhpUEGSXegx2CQYDBNyb290IGFscmVhZHkgZXhpc3RzOmhpEFNB5j8YhEBXAQBB9rRr4nBoDAEgStkwJATbMBNTQd8wuJpAVwEBNbgHAAB4EDAOeAMAEKXU6AAAADIfDBpUaGUgcHJpY2UgaXMgb3V0IG9mIHJhbmdlLjpBm/ZnznBoDAEQStkwJATbMHhTQeY/GIRAVwEAQfa0a+JwaAwBEErZMCQE2zBQQZJd6DFK2SEkBNshQFcGAXgJUDVPCAAAcGjYJh8MGmludmFsaWQgZG9tYWluIG5hbWUgZm9ybWF0OkH2tGvicQwBIErZMCQE2zBK2CYFRQwAaBHOStkwJATbMItyaWpQQZJd6DHYJhMMDnJvb3Qgbm90IGZvdW5kOgwBIUrZMCQE2zBK2CYFRQwAeErZMCQE2zA1VgUAAIt0aWxQQZJd6DFza9gmBAhAa0rZMCQE2zA3AQBK2UEkBNtBdUG3w4gDbRLOuEBXCwJ4CVA1lwcAAHBo2CYfDBppbnZhbGlkIGRvbWFpbiBuYW1lIGZvcm1hdDpBm/ZnznEMASBK2TAkBNswStgmBUUMAGgRzkrZMCQE2zCLcmlqUEGSXegx2CYTDA5yb290IG5vdCBmb3VuZDp5NQ4GAACqJhIMDWludmFsaWQgb3duZXI6eUH4J+yMqiYbDBZub3Qgd2l0bmVzc2VkIGJ5IG93bmVyOjV//v//c2tBw1qMvHhK2TAkBNswNWgEAAB0C3VpDAEhStkwJATbMErYJgVFDABsi1BBkl3oMXZu2CV5AAAAbkrZMCQE2zA3AQBK2UEkBNtBdwdBt8OIA28HEs4uBAlAbwcQznVpeErZMCQE2zBtEZtUNQMDAABpDAEiStkwJATbMErYJgVFDABsixFTQd8wuJp3CG8IQZwI7ZwmFm8IQfNUvx13CWlvCVBBL1jF7SLlIgppEVA1qQMAAAtBt8OIAwMALLFXBwAAAJ54eRS/dwppbG8KwkpRz9RTNScEAABpeErZMCQE2zB5EVQ1iQIAAG15eErZMCQE2zALVDX+AgAACEBXAwF4StgkBcoiBEUQAf8AMh8MGmludmFsaWQgZG9tYWluIG5hbWUgZm9ybWF0OjVN/f//cGhBw1qMvEGb9mfOcWl4StkwJATbMFA1NgMAAHJqEs4DACyxVwcAAACeahJR0GlqwkpRz9RQNXUDAABqEs5AVwICeErYJAXKIgRFEAH/ADIfDBppbnZhbGlkIGRvbWFpbiBuYW1lIGZvcm1hdDp52CQkeUH4J+yMqiYbDBZub3Qgd2l0bmVzc2VkIGJ5IGFkbWluOkGb9mfOcGh4StkwJATbMFA1qQIAAHFpEM5B+CfsjKomGwwWbm90IHdpdG5lc3NlZCBieSBvd25lcjp5aRNR0GhpwkpRz9RQNdACAABAVwQDeDXLBwAAStkwJATbMHAJcXlKEbMmC3o1KgUAAHEiVEoVsyYPeghQNbQEAADYqnEiQkoAELMmE3pK2CQFyiIERRAB/wC2cSIrSgAcsyYLejUGBgAAcSIcDBd1bnN1cHBvcnRlZCByZWNvcmQgdHlwZTpFaaomGAwTaW52YWxpZCByZWNvcmQgZGF0YTpBm/ZnznJqaFA11AEAAHNrNeD4//9qaHh5ehVVNZ4CAABAVwICeDUaBwAAStkwJATbMHBB9rRr4nFpaFA1oQEAAEVpaHh5VDU5AgAAQFcEAng17wYAAErZMCQE2zBwQZv2Z85xaWhQNXYBAAByajWC+P//aHh5UzWVAgAAc2lrUEEvWMXtQFcBAkH2tGvicGh4eRJUNRUHAABAVwMBeDWiBgAAStkwJATbMHBB9rRr4nFpaFA1KQEAAEVoeFA1HAIAAHJpahxTQd8wuJpAVwUEDAEBStkwJATbMErYJgVFDAB6i3AQcXhoUEGSXegxcmrYJAtqStkhJATbIXFpe55xaRAqDHhoUEEvWMXtIgt4aGlTQeY/GIR5Nb0AAABzDAECStkwJATbMErYJgVFDAB6i0rYJgVFDABri3R7EC4MeGxQQS9Yxe0iC3hseVNB5j8YhEBXAgR4eRF6VBTAcAwIVHJhbnNmZXJoUEGVAW9heTcCANgkJHgRentUFMBxeQwOb25ORVAxMVBheW1lbnQfaVRBYn1bUkVAVwEBeAwBAErZMCQE2zBQQZJd6DFwaErZISQE2yFAVwICDAEAStkwJATbMHB4NNNxeGhpeZ5TQeY/GIRAVwABeDcDAEBXAQJ5NPRweGhQNANAVwMCDAEhStkwJATbMErYJgVFDAB5i3B4aFBBkl3oMXFp2CYUDA90b2tlbiBub3QgZm91bmQ6aUrZMCQE2zA3AQBK2UEkBNtBcmo1m/b//2pAVwECeRHOStkwJATbMDWN////cHhoecJKUc/UUzQDQFcCAwwBIUrZMCQE2zBK2CYFRQwAeYtwejcEAHF4aGlTQeY/GIRAVwMEeXp7UzWFAAAAcHhoUEGSXegxcWnYJgtpStkoJATbKEBpStkwJATbMDcBAErZQSQE20FyahLOQFcDBXl6e1M0S3B8e3oTv3FpNwQAcnhoalNB5j8YhEBXAQIMASJK2TAkBNswStgmBUUMAHg15/7//4twaErYJgVFDAB5StkwJATbMDXQ/v//i0BXAQN4eVA0x3BoStgmBUUMAAwBAErZMCQE2zBKEHrQi0BXAAF42CQReErYJAXKIgRFEAAUsyIDCUBXAwA3BQBwaNgmHAwXZmFpbGVkIHRvIGdldCBjb21taXR0ZWU6aErYJAXKIgRFEHFpaRGfEqGfaFBBajPpCXJq2CQLakH4J+yMqiYfDBpub3Qgd2l0bmVzc2VkIGJ5IGNvbW1pdHRlZTpAVwMCAD9weSYFABBweErYJAXKIgRFEBAoD3hK2CQFyiIERRBoMgQJQHgQznF5JhVpAGEwCGkAerYiAwmqJgQJQCIKaTRAqiYECUARcmp4StgkBcoiBEUQEZ+1Jhh4as4ALSgMeGrONB2qJgQJQGqcciLceHhK2CQFyiIERRARn840A0BXAAF4AGEwB3gAejIQeAAwMAh4ADm2IgMJIgMIQFcDAnhK2CQFyiIERRBwaBMwCAH/AGguBAtAeAwBLlA3BgBxaUrYJAXKIgRFEHBoEi4EC0BoEjIIeaomBAtAaUrKEEtLMh1KcmlqzmpoEZ+zUDUP////qiYHRUVFC0CcIuNFRUVpQFcJAXhK2CQFyiIERRBwaBcwBh9oLgQJQHgMAS5QNwYAcWlK2CQFyiIERRAUKAQJQBTEIXJpSsoQS0szawAAAEpzEk1LznRsStgkBcoiBEUQECoHRUVFCUBsNwcAdW0QMAgB/wBtLg8MCm5vdCBhIGJ5dGU6bRAyDmwQzgAwKgdFRUUJQG0QKhRsStgkBcoiBEUQETIHRUVFCUBtamtR0JwjmP///0VFRWoQznZqEc53B2oTzncIbhApWAAAAG4aKFFuAH8oTG4B4AAuRm4BqQAqCm8HAf4AsyIDCSQ1bgGsACoPABBvBywJbwcAH7YiAwkkH24BwAAqCm8HAagAsyIDCSQObwgQKAlvCAH/ACoECUAIQFcMAXhK2CQFyiIERRBwaBIwBwAnaC4ECUB4DAE6UDcGAHFpStgkBcoiBEUQcGgTMAYYaC4ECUAJchjEIXNpSsoQS0sz5gAAAEp0Ek1LznVtStgkBcoiBEUQECtxAAAAbBAqHWkRzkrYJAXKIgRFEBAoB0VFRQlAEGtsUdAiTWxoEZ8qH2lsEZ/OStgkBcoiBEUQECgHRUVFCUAQaxdR0CIqaiYJRUVFCUAiIAhyGWifbJ52bHcHbwdutSYPEGtvB1HQbwecdwci7yJabUrYJAXKIgRFEBQyB0VFRQlAbQAQUDcIAHcIAv//AABvCC4iDBtmcmFnbWVudCBvdmVyZmxvd3MgdWludDE2OiBtizpsdwlqJglsGJ5on3cJbwhrbwlR0JwjHf///0VFRWgYLghqqiYECUBrEM53Cm8KAQAgMBdvCgECICgQbwoB/j8oCW8KAf8/MgQJQG8KAQEgKhdrEc53C28LAQACMAlvCwG4DSoECUAIQFcCAXgIUDUJ/f//cGjYJh8MGmludmFsaWQgZG9tYWluIG5hbWUgZm9ybWF0OmhK2CQFyiIERRBxeHhK2CQFyiIERRBoaRGfzkrYJAXKIgRFEGhpEp/OStgkBcoiBEUQnhGen0vKS5+MQFcFBHsQLhUMEGludmFsaWQgcmVkaXJlY3Q6eUrYJAXKIgRFEBAqEQwMaW52YWxpZCBuYW1lOnl5StgkBcoiBEUQEZ/OAC4qFHkQeUrYJAXKIgRFEBGfS5+MgXh5UDVpAAAAcAwAcWhBnAjtnCY3aEHzVL8dStlBJATbQXJqEc4SznNqEM5qEM5K2CQFyiIERRARn850bHoqBGtAbBUqBGtxIsVpDACXJhILStkwJATbMErZKCQE2yhAeGl6exGfVDVA////QFcCAnk1zf7//0rZMCQE2zBweGhQNVr5//9FaHlQNU36//9xeGkYU0HfMLiaQA==
    logger.go:146: 2024-11-14T15:52:24.389+0300	DEBUG	done processing headers	{"headerIndex": 11, "blockHeight": 10, "took": "252.292µs"}
    logger.go:146: 2024-11-14T15:52:24.390+0300	DEBUG	done processing headers	{"headerIndex": 12, "blockHeight": 11, "took": "228.792µs"}
    logger.go:146: 2024-11-14T15:52:24.391+0300	DEBUG	done processing headers	{"headerIndex": 13, "blockHeight": 12, "took": "223.833µs"}
    logger.go:146: 2024-11-14T15:52:24.392+0300	DEBUG	done processing headers	{"headerIndex": 14, "blockHeight": 13, "took": "214.166µs"}
    basic.go:227: NNS token #1 ID (hex): 6e656f2e636f6d
    logger.go:146: 2024-11-14T15:52:24.393+0300	DEBUG	done processing headers	{"headerIndex": 15, "blockHeight": 14, "took": "224.25µs"}
    logger.go:146: 2024-11-14T15:52:24.394+0300	DEBUG	done processing headers	{"headerIndex": 16, "blockHeight": 15, "took": "218.584µs"}
    basic.go:300: contract (../../examples/nft-d): 
        	Hash: 2f5c1826bb4da1c764a8871427e4044cf3e82dbd
        	AVM: VgIMFK7Y2ztdZkygS9177TeZL/lYVgtBStkwJATbMGBADARORlNPQBJAQfa0a+I0A0BXAwEQcAwBc0rZMCQE2zByeGpQQZJd6DFxadgkC2lK2SEkBNshcGhAVwEBDAFiStkwJATbMHBoStgmBUUMAHiLQFcBAng043BoStgmBUUMAHmLQFcBAQwBdErZMCQE2zBwaErYJgVFDAB4i0BXAQJ4NONwaErYJgVFDAB5i0BXAQEMAWlK2TAkBNswcGhK2CYFRQwAeItAVwUBeErYJAXKIgRFEAAUKBYMEWJhZCBvd25lciBhZGRyZXNzOkH2tGvicBBxaHhQNawAAAByakGcCO2cJiNqQfNUvx1K2TAkBNswc3hrUDVb////dGlobFA0CJ5xItlpQFcBAnh5UEGSXegxcGjYJAtoStkhJATbIUAQQFcCBHl6UDUo////cHhoUDTWcWl7nnFpEDINeGhpU0HmPxiEIgp4aFBBL1jF7WlAVwEBeErYJAXKIgRFEAAUKBYMEWJhZCBvd25lciBhZGRyZXNzOkH2tGvicGh4UDQDQFcCAnk1sf7//3B4aBNTQd8wuJpxaUBXBwN4StgkBcoiBEUQABQoGQwUaW52YWxpZCAndG8nIGFkZHJlc3M6QZv2Z85wC3EJcmh5UDVYAwAAc2tBnAjtnCYZaiYECUBrQfNUvx1K2TAkBNswcQhyIuNqqiYSDA11bmtub3duIHRva2VuOmlB+CfsjKomBAlAaXlQNTz+//90aGxQNer+//91aXZu2CYGeNgiAwkkIG7YJBl42CQVbkrZKCQE2yh4StkoJATbKJciAwkiAwiqJitoaXltm1Q1yv7//0VoeWlTNe8CAABoeHltVDW2/v//RWh5eFM1xQIAAGl4eW16FVU0BAhAVwIFeHl7elQUwHAMCFRyYW5zZmVyaFBBlQFvYXk3AADYJCR4e3p8VBTAcXkMDm9uTkVQMTFQYXltZW50H2lUQWJ9W1JFQFcFAUH2tGvicGh4UDWdAAAAqiYSDA11bmtub3duIHRva2VuOng1nP3//3FoaVBBkl3oMUrZMCQE2zByajcBAErZQSQE20FzaxHONwIADAhvYmplY3RJRGsQzjcCAAwLY29udGFpbmVySUQMDU5lb0ZTIE9iamVjdCB4NwIAiwwEbmFtZRO+dGxAVwMAQfa0a+JwDAFpStkwJATbMHFoaRNTQd8wuJpyakBXAgJ5NRT9//9weGhQQZJd6DFxadiqQFcFBXhK2CQFyiIERRAAFCgbDBZpbnZhbGlkICdmcm9tJyBhZGRyZXNzOnlK2CQFyiIERRAAFCgZDBRpbnZhbGlkICd0bycgYWRkcmVzczp6EC4WDBFuZWdhdGl2ZSAnYW1vdW50Jzp6AGQyFQwQaW52YWxpZCAnYW1vdW50JzpBm/ZnznBoe1A1Yf///6omEgwNdW5rbm93biB0b2tlbjp4Qfgn7IyqJgQJQHh7UDUW/P//cWhpUDXE/P//cnpqMgQJQHhza9gmBnnYIgMJJCBr2CQZedgkFWtK2SgkBNsoeUrZKCQE2yiXIgMJIgMIqiYzaHh7eptUNZ78//90bBAqC2h7eFM1vwAAAGh5e3pUNYb8//90bBAoC2h7eVM1kQAAAHh5e3p8FVU10P3//whAVwEBQfa0a+JwaHhQNaz+//+qJhIMDXVua25vd24gdG9rZW46aHhQNEFAVwICeErYJAXKIgRFEAAUKBcMEmJhZCBob2xkZXIgYWRkcmVzczpB9rRr4nB4eVA1Ovv//3FoaVA16Pv//0BXAgJ5NTn7//9weGgUU0HfMLiacWlAVwEDeXpQNTr7//9weGh6U0HmPxiEQFcBA3l6UDUk+///cHhoUEEvWMXtQFcOAzwGAgAAEgIAABFwQTlTbjxxaXJq2CYbDBTPduKL0AYsSkeO41VhARMZ88+k0tgiAwkkSmrYJEMMFM924ovQBixKR47jVWEBExnzz6TS2CQqakrZKCQE2ygMFM924ovQBixKR47jVWEBExnzz6TSStkoJATbKJciAwkiAwiqJhkMFG9ubHkgR0FTIGlzIGFjY2VwdGVkOnkCAMqaOy4nDCJtaW50aW5nIE5GU08gY29zdHMgYXQgbGVhc3QgMTAgR0FTOnpK2UAkBNtAc2tK2CQFyiIERRASKBMMDmludmFsaWQgJ2RhdGEnOmsQzkrZMCQE2zB0bErYJAXKIgRFEAAgKBkMFGludmFsaWQgY29udGFpbmVyIElEOmsRzkrZMCQE2zB1bUrYJAXKIgRFEAAgKBYMEWludmFsaWQgb2JqZWN0IElEOm1sEr92bjcDAHcHbwc3BAB3CEGb9mfOdwlvCW8IUDWn/P//JjQML05GU08gZm9yIHRoZSBzcGVjaWZpZWQgb2JqZWN0IGlzIGFscmVhZHkgbWludGVkOm8INYT5//93Cm8JbwpvB1NB5j8YhG8JNfH4//93C28Jbwh4UzUQ/v//bwl4bwgAZFQ16vn//0VvC5x3CwwBc0rZMCQE2zB3DG8JbwxvC1NB5j8YhAt4bwgAZAsVVTUi+///aNgkHj0cYRF3DQoXAAAANj0Qbw0kCAoLAAAANhB3DT9AVwEAWQthcGjYJBBoStkoJATbKEHP50eWOEBYQfgn7IxANPmqJhsMFm9ubHkgb3duZXIgY2FuIGRlc3Ryb3k6NwUAQFcAAjTUqiYaDBVvbmx5IG93bmVyIGNhbiB1cGRhdGU6eHlQNwYAQA==
    logger.go:146: 2024-11-14T15:52:24.430+0300	DEBUG	done processing headers	{"headerIndex": 17, "blockHeight": 16, "took": "250.166µs"}
    logger.go:146: 2024-11-14T15:52:24.431+0300	DEBUG	done processing headers	{"headerIndex": 18, "blockHeight": 17, "took": "227.917µs"}
    basic.go:260: NFSO token #1 ID (hex): 7e244ffd6aa85fb1579d2ed22e9b761ab62e3486
    logger.go:146: 2024-11-14T15:52:24.432+0300	DEBUG	done processing headers	{"headerIndex": 19, "blockHeight": 18, "took": "219.75µs"}
    logger.go:146: 2024-11-14T15:52:24.432+0300	DEBUG	done processing headers	{"headerIndex": 20, "blockHeight": 19, "took": "217.75µs"}
    logger.go:146: 2024-11-14T15:52:24.433+0300	DEBUG	done processing headers	{"headerIndex": 21, "blockHeight": 20, "took": "210.5µs"}
    basic.go:300: contract (../../internal/basicchain/testdata/storage/storage_contract.go): 
        	Hash: ebc0c16a76c808cd4dde6bcc063f09e45e331ec7
        	AVM: VgEMAQFK2TAkBNswYEBXAwJ5qiY3QZv2Z85wEHFpAf8AtSYoWErYJgVFDAAMAQBK2TAkBNswShBp0ItyaGppU0HmPxiEaZxxItVAVwEAQZv2Z85waFgUU0HfMLiaQA==
    logger.go:146: 2024-11-14T15:52:24.466+0300	DEBUG	done processing headers	{"headerIndex": 22, "blockHeight": 21, "took": "262.958µs"}
    logger.go:146: 2024-11-14T15:52:24.469+0300	DEBUG	done processing headers	{"headerIndex": 23, "blockHeight": 22, "took": "551.583µs"}
    logger.go:146: 2024-11-14T15:52:24.469+0300	WARN	contract invocation failed	{"tx": "82279bfe9bada282ca0f8cb8e0bb124b921af36f00c69a518320322c6f4fef60", "block": 23, "error": "at instruction 0 (ABORT): ABORT"}
    basic.go:280: FAULTed transaction:
        	hash LE: 82279bfe9bada282ca0f8cb8e0bb124b921af36f00c69a518320322c6f4fef60
        	block index: 23
    basic.go:300: contract (../../internal/basicchain/testdata/invoke/invokescript_contract.go): 
        	Hash: fe1a3e8f43f4ca945f2c907d94b7caf7489de35f
        	AVM: VwIADBQBDAMOBQYMDQIODw0DDgcJAAAAAErZMCQE2zBwaEH4J+yMqiYEEUAMFA0PAwIJAAIBAwcDBAUCAQAOBgwJStkwJATbMHFpQfgn7IyqJgQSQBNA
    basic.go:291: sendrawtransaction: 
        	base64: AB0AAACWP5gAAAAAAEDaEgAAAAAAGAAAAAHunqIsJ+NL0BSPxBCOCPdOj1BIsoAAXgsDAOh2SBcAAAAMFBEmW7QXJQBBvgTo+iQOOPV8HlabDBTunqIsJ+NL0BSPxBCOCPdOj1BIshTAHwwIdHJhbnNmZXIMFPVj6kC8KD1NDgXEjqMFs/Kgc0DvQWJ9W1IBQgxAJ6norhWoZxp+Hj1JFhi+Z3qI9DUkLSbfsbaLSaJIqxTfdmPbNFDVK1G+oa+LWmpRp/bj9+QZM7yC+S6HXUI7rigMIQKzYiv0AXvf4xfFiu1fTHU/IGt9uJYEb6fXdLvEv3+NwkFW57Mn
        	Hash LE: c11861dec1dd0f188608b725095041fcfc90abe51eea044993f122f22472753e
    basic.go:295: Block #20 stateroot LE: 9a431f425932ffb5c0a9b569fae2d5bc086753de21b6a83d6a41b46aaf21d014
    logger.go:146: 2024-11-14T15:52:24.505+0300	INFO	persisted to disk	{"blocks": 23, "keys": 1268, "headerHeight": 23, "blockHeight": 23, "took": "267.125µs"}
--- PASS: TestCreateBasicChain (0.32s)
PASS

Process finished with the exit code 0

```